esp-hal/CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Reserve `esp32` ROM stacks to prevent the trashing of dram2 section (#1289)
 - Fixing `esp-wifi` + `TRNG` issue on `ESP32-S2` (#1272)
 - Fixed core1 startup using the wrong stack on the esp32 and esp32s3 (#1286).
+- Fixed LCD_CAM i8080 potentially sending garbage to display (#1301)
+- ESP32: Apply fix for Errata 3.6 in all the places necessary. (#1315)
+- ESP32 & ESP32-S2: Fix I²C frequency (#1306)
+- Fixed LCD_CAM i8080 potentially sending garbage to display (#1301)
```

Comment:
Seems like this line insists on duplicating itself

Reply:
I rebased, it went strange (but then I'm not a git expert), and now I see it doesn't compile in the build. Maybe I did something wrong.

But then I tried taking main and building the i8080 example and It doesn't compile either on my machine, so maybe it's not me but rather something that changed on the main branch?

I could try fixing the example but I don't want to make a mess.

Reply:
I just tried compile the example on current main and it compiled fine for me 🤔